### ISSUE TITLE
some time when click on start i am getting these error in console of chrome browser 

### ISSUE THREAD

**Issue by TrRavi** [syl22-00/pocketsphinx.js#132]
Some time when i click on start button i am getting this 

`ERROR: "pocketsphinx.c", line 473: Cannot change search while decoding, end utterance first`

after this it does not work.. 
please let me know how to solve this.

**Comment by syl22-00** [comment]
@TrRavi , yes, you need to stop before you start a new decoding. We have not prevented this in the demo app.
